dbt/include/athena/macros/adapters/relation.sql

```diff
@@ -1,16 +1,10 @@
 {% macro athena__drop_relation(relation) -%}
-  {% set rel_type = adapter.get_table_type(relation.schema, relation.table) %}
+  {% set rel_type = relation.type %}
```

Comment:
This is different than get_table_type! get_table_type allows to get real glue table type in the catalog. relation.type comes from dbt, therefore all edge cases are not covered. Please revert back.

Reply:
The reason I changed this is the original `rel_type == 'table' ` wasn't being triggered. I printed out the `rel_type` parameter value and it was `TableType.TABLE ` which didn't return `True` on any Jinja type checks so I wasn't sure what the condition should be. What am I missing?

Reply:
ah I see, there was some time ago a refactor and TableType return enums.
You can consider to call explicitly `adapter.get_table_type(relation.schema, relation.table).value` 

e.g.
```
class TableType(Enum):
    TABLE = "table"
    VIEW = "view"
    CTE = "cte"
    MATERIALIZED_VIEW = "materializedview"
    ICEBERG = "iceberg_table"

print(TableType.TABLE.value)
```
so we should add either:
1. add `{% set rel_type = adapter.get_table_type(relation.schema, relation.table).value %}`
2. return `_type.value` in here https://github.com/dbt-athena/dbt-athena/blob/58d09c3b6608af741d499953dbb0ad47764087a9/dbt/adapters/athena/impl.py#L529 _type.value

@svdimchenko any idea about this behavior? I think that when working with Python the value property is called automatically, but when calling the enum from jinjia somehow the value property is not rendered invoked, therefore the first option might be better in this case.

Reply:
yeah, the first option is definitely the most optimal way to get table type as a string